shared/protocol/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -0,0 +1,486 @@
+/**
```

Comment:
Not sure why it is showing as new file. This probably means you'll have  to deal with a merge conflict.